src/libcollections/btree/map.rs

```diff
@@ -326,6 +326,19 @@ pub enum Entry<'a, K: 'a, V: 'a> {
              OccupiedEntry<'a, K, V>),
 }
 
+impl<'a, K: 'a + Debug + Ord, V: 'a + Debug> Debug for Entry<'a, K, V> {
```

Comment:
Per @alexcrichton's comments [here](https://github.com/rust-lang/rust/pull/33856#discussion_r64598872), these can be stable.